src/components/Welcome.js

```diff
@@ -1,8 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import TokenForm from './TokenForm';
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
I love this function. I think it would be nice to export this `useStyles` function from the theme, then just import it and use in the component, rather than repeating it in every file.

So in the component files, you would just need the import statement, and the `const classes = useStyles()`

Reply:
Yes! Thank you. I was trying to wrap my head around how to refactor and just couldn't get there. Thank you for the suggestion! It's way more DRY this way.